.github/workflows/playwright.yml

```diff
@@ -15,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: mcr.microsoft.com/playwright:v1.41.2-jammy
-    if: github.event.deployment_status.state == 'success'
+    if: true == false
```

Comment:
I highly recommend adding a comment here explaining why you are doing this, even if it's temporary, and when this change will be reverted.

Reply:
Updated here: 3bf3a66362874d1d290b46327047781ae266b0f7

Reply:
You should also create an issue in this repo to go along with that TODO and mention the issue number in this comment (that is why we have this checkbox item in our PR template: "All future TODOs are captured in issues, which are referenced in code comments.")

Reply:
The issue is being tracked in JIRA, not Github. 

The associated draft PR is found here: https://github.com/cisagov/XFD/pull/372 

...

Reply:
Excellent, that works! 👍

Reply:
See commit: 704398f220fa676c3e6f5533c808a4d05e511b93